portality/static/js/formulaic.js

```diff
@@ -1382,16 +1382,17 @@ var formulaic = {
             this.init = function() {
                 var viewClass = edges.css_classes(this.ns, "view");
                 var closeClass = edges.css_classes(this.ns, "close");
+                var viewDivClass = edges.css_classes(this.ns, "viewDiv")
+                this.divs = $("div[name='" + this.fieldDef["name"] + "__group']").find("textarea");
 
-                this.divs = $("div[name='" + this.fieldDef["name"] + "__group']");
                 for (var i = 0; i < this.divs.length; i++) {
                     var container = $(this.divs[i]);
                     var modalId = "modal-" + this.fieldDef["name"] + "-" + i;
 
                     var date = $("#" + this.fieldDef["name"] + "-" + i + "-note_date");
                     var note = $("#" + this.fieldDef["name"] + "-" + i + "-note");
 
-                    container.append(`<div><a href="#" class="` + viewClass + `">view note</a>
+                    $(`<button class="button button--cta ` + viewClass + `" style="margin: 0 1rem 1rem 0;">View note</button>
```

Comment:
@amdomanska I’ve made some slight modifications to the styling here (explained in here: https://github.com/DOAJ/doajPM/issues/2904#issuecomment-1008761062)